nnoir-onnx/nnoir_onnx/onnx.py

```diff
@@ -165,9 +165,12 @@ def _has_input(self, name):
     def to_NNOIR(self):
         inputs = list(map(lambda x: x.name, self.sess.get_inputs()))
         outputs = list(map(lambda x: x.name, self.sess.get_outputs()))
-        functions = self._to_NNOIR_functions()
-        nodes = [Value(n, self.nodes[n])
-                 for n in set(chain.from_iterable(map(lambda x: x.inputs + x.outputs, functions)))]
+        functions, new_values = self._to_NNOIR_functions()
+
+        nodes = new_values
+        for n in set(chain.from_iterable(map(lambda x: x.inputs + x.outputs, functions))):
+            if n in self.nodes:
+                nodes.append(Value(n, self.nodes[n]))
```

Comment:
We have to change here to append new intermediate values during conversions.

We hadn't append new `Values` during conversion before, then only inputs and outputs would be `Value`s.
`x.inputs` and `x.outputs` could be found in values.

After my PR, `n in self.nodes` will be failed due to new intermediate values.

Reply:
Why not use the follow functions to get new intermediate values?

https://github.com/Idein/nnoir/blob/750d27a732ccac735fdef33c34d29d7e0e6f3ee3/nnoir-onnx/nnoir_onnx/operators/utils.py#L63-L71

Reply:
Thanks you for your suggestion!
Maybe do I need to reflect the result of `gen_unregistered_node_name` using `register_node`?

Intermediate nodes are not constants: just have its shape and dtype.
So I cannot pass `val` in the argument of `register_node`.

Maybe I have to extend env?

Reply:
Please refer https://github.com/Idein/nnoir/blob/master/nnoir-onnx/nnoir_onnx/operators/lstm.py .

Reply:
Modified in https://github.com/Idein/nnoir/pull/101/commits/3da22eb94585a3dbb323ef6a5d0c11769c57fd5d#diff-7a2e3dc41912ced52704ec4734b2305a16a70dc8747c436417a833222d73fa2fR65-R71